### PR TITLE
Kops - Force AMD64 for Kubernetes presubmit

### PR DESCRIPTION
**Pull request by hakman**
xRef: https://github.com/kubernetes/kops/issues/9526
xRef: https://github.com/kubernetes/kops/pull/9535

/cc @rifelpet @jqmichael @BenTheElder

### CODE REVIEW

File: config/jobs/kubernetes/kops/kubernetes-presubmits.yaml [hunk 1]
```diff
@@ -36,6 +36,7 @@ presubmits:
         - --aws-cluster-domain=test-cncf-aws.k8s.io
         - --build=bazel
         - --cluster=
+        - --env=KOPS_ARCH=amd64
```
Comment (jqmichael) [inline config/jobs/kubernetes/kops/kubernetes-presubmits.yaml:39]: @hakman 
I assume this will take effect once Kops cut a new 1.19 release?

Comment (hakman) [reply config/jobs/kubernetes/kops/kubernetes-presubmits.yaml:39]: Should work now if you re-run the job.

**Comment by BenTheElder** [comment]
thank you! :pray:

**Comment by hakman** [comment]
np. thanks for pointing me in the right direction :)

**Pull request merged**